source/agora/node/Ledger.d

```diff
@@ -722,18 +722,19 @@ public class Ledger
         a node itself
 
         Params:
+            height = the heigh for which we want to check self slashing
             data = consensus data
 
         Returns:
-            true if the consensus data has the information that is slashing
-            a node itself.
+            error string, if node is trying to slash itself, null otherwise
 
     ***************************************************************************/
 
-    public bool isSelfSlashing (in Height height, in ConsensusData data) @safe nothrow
+    public string isSelfSlashing (in Height height, in ConsensusData data) @safe nothrow
```

Comment:
see https://github.com/bosagora/agora/pull/2244/files#r664243457 for why it is not in a separate commit